formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -12,6 +12,12 @@ import kotlinx.serialization.internal.*
 import kotlinx.serialization.modules.*
 import kotlin.experimental.*
 
+private const val ADD_DATA_ONE_BYTE = 24
+private const val ADD_DATA_TWO_BYTES = 25
+private const val ADD_DATA_FOUR_BYTES = 26
+private const val ADD_DATA_EIGHT_BYTES = 27
+private const val ADD_DATA_INDEF_LENGTH = 31
```

Comment:
Not sure how these are defined in the RFC, so feel free to disregard my suggestion if it doesn't make sense here (i.e. if RFC lists as numbers, not binary, then disregard).

```suggestion
private const val ADD_DATA_ONE_BYTE = 0b0001_1000 // 24
private const val ADD_DATA_TWO_BYTES = 0b0001_1001 // 25
private const val ADD_DATA_FOUR_BYTES = 0b0001_1010 // 26
private const val ADD_DATA_EIGHT_BYTES = 0b0001_1011 // 27
private const val ADD_DATA_INDEF_LENGTH = 0b0001_1111 // 31
```

Reply:
RFC does list these values in the decimal format. Good idea though.

Reply:
I just saw that they used the binary representation elsewhere in this class, but if the RFC lists them as numbers, then I think it's best to stick with the "documentation" (the RFC) for others that may read the code.